### PR TITLE
Clean Code initial documentation template

### PR DESCRIPTION
**Pull request by jstevenson**
- [n/a] Related GitHub issue(s) linked in PR description
- [n/a] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [n/a] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [n/a] Automated tests are passing
- [n/a] No _decreases_ in automated test coverage
- [n/a] Documentation updated (readme, docs, comments, etc.)
- [n/a] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

File: CLEAN-CODE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Clean Code
+
+These are Andculture's code standards based on Robert Martin's _Clean Code_.
+
+## Meaningful Names
+
+### Make Meaningful Distinctions
```
Comment (brandongregoryscott) [inline CLEAN-CODE.md:7]: Are these all meant to be different heading sizes? I'm not sure by looking at the rich preview.

File: CLEAN-CODE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+| Bad                                            | Good                                                         |
+| ---------------------------------------------- | ------------------------------------------------------------ |
+| Product<br>ProductData<br>ProductInfo          | ProductDetail<br>ProductListItem<br>HomePageProduct          |
+| GetActiveAccount()<br>GetActiveAccountRecord() | GetActiveAccountFullDetails()<br>GetActiveAccountForHeader() |
+
+#### Names Should have Meaningful Distinctions
+
+Being able to differentiate classes, methods or variables isn't enough, they should have meaningful
+distinctions. This means that we shouldn't just iterate a number over items. The names should show
+why they are different than the other similar objects.
```
Comment (brandongregoryscott) [inline CLEAN-CODE.md:34]: The only thing I might add here is a blurb about seeding multiple entities for a test scenario - if there are multiple instances but the values don't need to be read back (ie, just verifying that an `Index` endpoint returns >1 record), consider just putting them in a list without saving out individual variables. Another scenario might be seeding two entities that differ - naming one `expectedUser` and the other `unexpectedUser` help the reader understand why they are being setup differently.

**Review by brandongregoryscott — APPROVED** [review]
Left a few thoughts, but it looks good 👍

File: CLEAN-CODE.md (hunk 1 above)
Comment (jstevenson) [reply CLEAN-CODE.md:7]: Yes. VS Code's preview is pretty bad, but you can see the differences at [CLEAN-CODE.md](https://github.com/AndcultureCode/AndcultureCode/blob/fb7d5016d3ba52b6b5672251174541ea78bd2b94/CLEAN-CODE.md).

`##` corresponds with chapters, `###` with section headers. and `####` shows small topics within the section.

File: CLEAN-CODE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+These are Andculture's code standards based on Robert Martin's _Clean Code_.
+
+## Meaningful Names
+
+### Make Meaningful Distinctions
+
+#### Avoid Noise Words
+
+Noise words are words that don't provide additional information, and are often added to
```
Comment (wintondeshong) [inline CLEAN-CODE.md:11]: Wondering if we should take into account later commentary from Martin regarding use of words that are patterns known by engineers (ie. Record). Not sure if folks feel it is worth mention about the exceptions or we are safe to leave that assumed.

**Review by wintondeshong — APPROVED** [review]
@jstevenson thanks for putting this together. just left a question. otherwise looks good to me

File: CLEAN-CODE.md (hunk 2 above)
Comment (jstevenson) [reply CLEAN-CODE.md:34]: Good example, I will push some additional changes in a minute.

File: CLEAN-CODE.md (hunk 3 above)
Comment (jstevenson) [reply CLEAN-CODE.md:11]: Good point, I will drop Record and add some text.

**Pull request merged**